lib/modules/datasource/index.ts

```diff
@@ -428,12 +428,13 @@ export function supportsDigests(datasource: string | undefined): boolean {
   return !!ds && 'getDigest' in ds;
 }
 
-function getDigestConfig(
+export function getDigestConfig(
```

Comment:
please don't export this, as it's internal API. can't we test it without making public?